apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -609,7 +609,16 @@ function main() {
     });
 
     $scope.print = function () {
-      $window.print();
+      let content = document.getElementsByClassName('all_cards')[0].innerHTML;
+      $window.frames["print_frame"].document.body.innerHTML= content;
+
+      //Want to apply this only to the printing frame, so add here rather than inline
+      let cards = $window.frames["print_frame"].document.getElementsByClassName('signin_card');
+      for (let i = 0; i < cards.length; i++){
+        cards[i].style.width = '300px';
+      }
+      $window.frames["print_frame"].window.focus();
+      $window.frames["print_frame"].window.print();
```

Comment:
nit: I think all instances of `frames['print_frame']` can/should be `frames.print_frame` instead. A little surprised our linter doesnt complain about this.

Reply:
nit: content/cards should be const rather than let. Making `cards` const might feel a little strange, because we are changing aspects of the underlying object (style.width), but the variable that points to our object is not changing, and thus const is valid.